### PR TITLE
Proof-reading on tut - PoE

### PR DESCRIPTION
**Pull request by jimmychu0807**
@sacha-l @NukeManDan 

Proofreading on tut - PoE. Also removed the testing section in `_redirect` file.

I have updated the corresponding:
- [tutorial solution in Node Template](https://github.com/substrate-developer-hub/substrate-node-template/tree/tut-sol/proof-of-existence%2Flatest)
- [tutorial solution in Front-end Template](https://github.com/substrate-developer-hub/substrate-front-end-template/tree/tut-sol/proof-of-existence%2Flatest)

### CODE REVIEW

File: v3/tutorials/02-proof-of-existence/index.mdx [hunk 1]
```diff
@@ -315,87 +317,99 @@ Our pallet will only emit an event in two circumstances:
 1. When a new proof is added to the blockchain.
 2. When a proof is removed.
 
-The events can contain some additional data, in this case, each event will also display who
-triggered the event (`AccountId`), and the proof data (as `Vec<u8>`) that is being stored or
-removed. Note that convention is to include an array with descriptive names for these parameters at
-the end of event documentation.
+Events can contain some additional data. In this case, each event will also display who trigger the
```
Comment (sacha-l) [inline v3/tutorials/02-proof-of-existence/index.mdx:320]:
```suggestion
Events can contain some additional data. In this case, each event will also display who triggers the
```

File: v3/tutorials/02-proof-of-existence/index.mdx [hunk 2]
```diff
@@ -315,87 +317,99 @@ Our pallet will only emit an event in two circumstances:
...
+event (`AccountId`), and the proof data (as `Vec<u8>`) that is being stored or removed. Note that
+convention is to include an array with descriptive names for these parameters at the end of event
```
Comment (sacha-l) [inline v3/tutorials/02-proof-of-existence/index.mdx:322]:
```suggestion
convention is to include an array with descriptive names for these parameters when documenting
```

File: v3/tutorials/02-proof-of-existence/index.mdx [hunk 3]
```diff
@@ -315,87 +317,99 @@ Our pallet will only emit an event in two circumstances:
...
+documentation.
```
Comment (sacha-l) [inline v3/tutorials/02-proof-of-existence/index.mdx:323]:
```suggestion
an event.
```

File: v3/tutorials/02-proof-of-existence/index.mdx [hunk 4]
```diff
@@ -546,7 +556,7 @@ yarn install
 yarn start
 ```
 
-A new tab should open in your web browser and you'll see the following interface.
+A new tab should open in your web browser and you'll the following interface.
```
Comment (sacha-l) [inline v3/tutorials/02-proof-of-existence/index.mdx:559]:
```suggestion
A new tab should open in your web browser and you should see the following interface.
```

**Review by sacha-l — COMMENTED** [review]
Minor edits ✏️

**Comment by jimmychu0807** [comment]
Thank you for reviewing.

**Pull request merged**